src/lib_gui/qt/element/code/QtCodeArea.cpp

```diff
@@ -804,6 +833,29 @@ void QtCodeArea::setNewTextCursor(const QTextCursor& cursor)
 	verticalScrollBar()->setValue(verticalValue);
 }
 
+void QtCodeArea::dragSelectedText()
+{
+	QMimeData* mimeData = new QMimeData;
+	const QString text(textCursor().selection().toPlainText());
+	mimeData->setText(text);
+
+	QDrag* drag = new QDrag(this);
+	drag->setMimeData(mimeData);
+	drag->exec(Qt::CopyAction);
+
+	m_isDragging = false;
```

Comment:
Setting `m_isDragging = false;` to avoid that this method is constantly called should better be placed in line 692 within `QtCodeArea::mouseMoveEvent`.

Reply:
if I clear flag at line 692 as you suggested then it will be cleared in the between two `mouseMoveEvent` events with small `delta` distance (which is almost always) and drag will not be working properly (you should drag mouse really fast for it to work)
Although I did move `m_isDragging = false` to the line 692 which is more suitable place for it